docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -323,7 +323,153 @@ Use this to produce a PDF of the band structure:
            ...: bands = load_node(<PK>)  # PK of the `band_structure` node
            ...: bands.show_mpl(y_origin=fermi_energy, plot_zero_axis=True)
 
+Querying the database
+---------------------
+
+As you will use AiiDA to run your calculations, the database that stores all the data and the provenance will quickly grow to be very large.
+To help you find the needle that you might be looking for in this big haystack, we need an efficient search tool.
+AiiDA provides a tool to do exactly this: the ``QueryBuilder``.
+The ``QueryBuilder`` acts as the gatekeeper to your database, to whom you can ask questions about its contents (also referred to as queries), by specifying what are looking for.
+In this final part of the tutorial, we will show an short demo on how to use the ``QueryBuilder`` to make these queries and understand/use the results.
+
+First, we'll import an archive of calculations performed on a group of 57 perovskite structures, using three different functionals: LDA, PBE and PBEsol:
```

Comment:
I think this might be sufficient; it could be nice if we had more "content" details but I'm not sure the specification of the functionals is of relevance to introduce the DB (at least not to what I had in mind).

```suggestion
First, we'll import an archive of a study performed on a group of 57 different perovskites:
```